modules/test/playwright/tests/style-book-web/previewStyleBookOnDisplayPageTemplates.spec.ts

```diff
@@ -0,0 +1,247 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {pageEditorPagesTest} from '../../fixtures/pageEditorPagesTest';
+import {styleBookPageTest} from '../../fixtures/styleBookPageTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import getRandomString from '../../utils/getRandomString';
+import getBasicWebContentStructureId from '../../utils/structured-content/getBasicWebContentStructureId';
+import {blogsPagesTest} from '../blogs-web/fixtures/blogsPagesTest';
+
+const test = mergeTests(
+	apiHelpersTest,
+	blogsPagesTest,
+	isolatedSiteTest,
+	loginTest(),
+	pageEditorPagesTest,
+	styleBookPageTest,
+	workflowPagesTest
+);
+
+test('Add a heading fragment and view the display page templates is shown in preview type selector', async ({
+	apiHelpers,
+	displayPageTemplatesPage,
+	page,
+	pageEditorPage,
+	site,
+	styleBooksPage,
+}) => {
+	const displayPageTemplateNameBlogsEntry = getRandomString();
+	const displayPageTemplateNameWebContent = getRandomString();
+
+	const styleBookName = getRandomString();
+
+	const placeHeadingFragmentAndSelectText = async () => {
+		await pageEditorPage.addFragment('Basic Components', 'Heading');
+
+		await page.getByLabel('Browser').click();
+		await page.getByLabel('Select Heading').click();
+		await page.getByLabel('Select element-text').click();
```

Comment:
I remember we discussing about adding blank lines between statements and we came up to the conclusion to follow the backend pattern. That is, to add a new line between statements when the order matters. 

Please, check other places like this